### PR TITLE
[Feature][CDC] Support write cdc changelog event in doris sink

### PR DESCRIPTION
**Pull request by john8628**
…s sink

## Purpose of this pull request

#3726 
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by hailin0** [comment]
waiting for this pr merged and add e2e testcase

https://github.com/apache/incubator-seatunnel/pull/3865

File: seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/serialize/DorisColumnRowSerialize.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    private DorisRecord serializeDelete(SeaTunnelRow row) {
+        SerializationSchema serializer = createSerializer(sinkConfig, rowType);
+        String record = new String(serializer.serialize(row));
+        DorisRecord dorisRecord = DorisRecord.builder().record(record)
+            .enableDelete(true).mergeType("DELETE").build();
+        return dorisRecord;
+    }
+
+    private DorisRecord serializeUpsert(SeaTunnelRow row) {
+        SerializationSchema serializer = createSerializer(sinkConfig, rowType);
```
Comment (TaoZex) [inline seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/serialize/DorisColumnRowSerialize.java]: Calling the createSerializer method on each row may affect performance.
How about calling the createSerializer method only once?

Comment (john8628) [reply seatunnel-connectors-v2/connector-doris/src/main/java/org/apache/seatunnel/connectors/doris/serialize/DorisColumnRowSerialize.java]: OK， that's better ;

**Comment by hailin0** [comment]
> waiting for this pr merged and add e2e testcase
> 
> #3865

Please add e2e testcase
@john8628

**Comment by hailin0** [comment]
reference this pr add e2e
@john8628 
https://github.com/apache/incubator-seatunnel/pull/3865

**Comment by john8628** [comment]
> 

ok , thanks,i will work on it recently~

**Pull request closed without merging**